modelplace_api/visualization.py

```diff
@@ -67,6 +67,37 @@ def draw_line(
             i += 1
 
 
+def draw_text_label(
+    image: np.ndarray,
+    detection: AgeGenderLabel,
+    text: str,
+    font_scale: float = 0.6,
+    thickness: int = 1,
+    bg_color: tuple = (128, 128, 128),
```

Comment:
Probably, usage of such parameters should be consistent: either use color from RGB_COLORS or manually coded RGB color in all appearances. However, I believe we won't find the exact (128, 128, 128) color in our array.

@lxdv, what do you think?

Reply:
Agree. Looks like we could also create one more array with not just palette colors but rather with specific design colors.

At the moment, @AlexanderSlav decided to use `bg_color: tuple = RGBA_COLORS[196]` so let's use it for now